linera-base/src/data_types.rs

```diff
@@ -172,6 +172,40 @@ pub struct Resources {
     // TODO(#1533): Allow declaring calls to other applications instead of having to count them here.
 }
 
+/// A message that is scheduled to be sent.
+#[derive(Clone, Debug, Deserialize, Serialize)]
+#[cfg_attr(with_testing, derive(Eq, PartialEq))]
+pub struct OutgoingMessage<Message> {
```

Comment:
I wonder whether we should give it a different name than `linera-chain::data_types::OutgoingMessage`.
E.g. this one could be `PostedMessage` (as in: the application posted it, but it's not quite in the chain's outbox yet)?
Or we could call it `SdkOutgoingMessage`, and then in `linera-sdk` import it `as OutgoingMessage`?
Not sure what's best. Can also be changed in a later PR, if we decide to!

Reply:
How about `SendMessageRequest`? That might be more accurate in this context.